packages/tests/tests/relay.node.spec.ts

```diff
@@ -90,11 +89,11 @@ describe("Waku Relay [node only]", () => {
 
     it("Subscribe", async function () {
       log("Getting subscribers");
-      const subscribers1 = waku1.libp2p.pubsub
-        .getSubscribers(DefaultPubSubTopic)
+      const subscribers1 = waku1.libp2p.services
+        .pubsub!.getSubscribers(DefaultPubSubTopic)
```

Comment:
I think we had this discussion in the past but I don't remember the result. Do we ever expect users to try to access the `pubsub` property?


If so, then the change of API is an issue as it's much more cumbersome now.

Reply:
I think you refer to the discussion about `gossipSub` for which I made it [public](https://github.com/waku-org/js-waku/blob/376bcf2a0a25bff27bc7eb04d2414ccfa9166fb5/packages/relay/src/index.ts#L53) and we have a debugging method as well.

But I wouldn't expect developers to access it directly. Additionally, this is the change coming from the new design of `libp2p`, to mitigate we can write a proxy but I don't think it is worth the hassle for now.